cms/templates/partials/hero.html

```diff
@@ -65,13 +65,12 @@ <h4>
       </div>
       <div class="col-lg-5">
         {% if page.video_url %}
-          {% with page.video_url|get_video_id as video_id %}
```

Comment:
tag `get_video_id` is no needed any more. you can remove it.

Reply:
it is already removed.